fastlane/download_metadata.swift

```diff
@@ -141,9 +157,16 @@ func downloadTranslation(
                 try FileManager.default.removeItem(at: URL(fileURLWithPath: releaseNotesPath))
             }
 
-            try subtitle?.write(toFile: "\(languageFolder)/subtitle.txt", atomically: true, encoding: .utf8)
+            if shouldUpdateSubtitle(locale: languageCode) {
+                try subtitle?.write(toFile: "\(languageFolder)/subtitle.txt", atomically: true, encoding: .utf8)
+            }
+
             try whatsNew?.write(toFile: "\(languageFolder)/release_notes.txt", atomically: true, encoding: .utf8)
```

Comment:
By the same token as the comment above, I decided to not add methods to opt-out of updating the release notes and store description because they are not necessary for the time being. YAGNI.